app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,20 @@
+class SessionsController < ApplicationController
+  def new; end
+
+  def create
+    user = User.find_by email: params[:session][:email]
+    if user&.authenticate params[:session][:password]
+      flash[:success] = t "authentication.messages.login_success"
+      log_in user
+      redirect_to user
+    else
+      flash[:danger] ||= t "authentication.messages.incorrect_email_password"
```

Comment:
```suggestion
      flash[:danger] = t "authentication.messages.incorrect_email_password"
```
Bạn thử xem còn bị rubocop bắt nữa ko

Reply:
Không bị nữa rồi b. 
Nãy xoá code gán flash[:danger] phía trên mà quên sửa dấu ||
Tks b nhé